.github/workflows/bazel.yml

```diff
@@ -126,14 +129,20 @@ jobs:
         with:
           bazelisk-cache: true
           bazelrc: common --color=yes
+          # Workaround for long path issues: https://github.com/bazelbuild/bazel/pull/22532
+          output-base: ${{ contains(inputs.os, 'windows') && 'D://b' || '' }}
```

Comment:
CI is failing on the windows-11-arm runner. I think this line is causing it because the windows arm runners don't have a D drive: https://github.com/actions/partner-runner-images/issues/87